src/styled/styledBalanceForm.tsx

```diff
@@ -49,14 +47,31 @@ const Button = styled.button`
   }
   margin-left: auto;
   margin-top: 3rem;
+  @media (max-width: 768px) {
```

Comment:
We don't want to have @media in random places.
Let's make a *rule* that we use media only in layout components and in global page object (where it should only modify rem size).

Reply:
@matiop4 what do you think about this rule?